<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="tbServer">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{
	border:none;
}
QToolButton:pressed
{
	background:#80eaea;
}</string>
          </property>
          <property name="text">
           <string>服务状态</string>
          </property>
          <property name="icon">
           <iconset resource="image/resource.qrc">
            <normaloff>:/new/images/servermonitor.png</normaloff>:/new/images/servermonitor.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbDeviceState">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{
	border:none;
}
QToolButton:pressed
{
	background:#80eaea;
}</string>
          </property>
          <property name="text">
           <string>设备状态</string>
          </property>
          <property name="icon">
           <iconset resource="image/resource.qrc">
            <normaloff>:/new/images/device.png</normaloff>:/new/images/device.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbOnlineUser">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>在线用户</string>
          </property>
          <property name="icon">
           <iconset resource="image/resource.qrc">
            <normaloff>:/new/images/users.png</normaloff>:/new/images/users.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbSystem">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>系统设置</string>
          </property>
          <property name="icon">
           <iconset resource="image/resource.qrc">
            <normaloff>:/new/images/system.png</normaloff>:/new/images/system.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lbDatetime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="swContainter">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page"/>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image/resource.qrc"/>
 </resources>
 <connections/>
</ui>
